src/common/transformations/src/transformations/op_conversions/unique_decomposition.cpp

```diff
@@ -69,6 +69,27 @@ ov::pass::UniqueDecomposition::UniqueDecomposition() {
             return false;
         }
 
+        // in case the input with the single input, there is no transformation required
+        // this separate pass needed since now there is no handler for empty tensors
+        // that should be converted to empty Constant nodes
+        // x is already of rank equal to 1 after Reshape
```

Comment:
x already has rank equal to 1 after Reshape

Reply:
it is nit. I will fix it in the next PR.